src/mame/drivers/photoply.cpp

```diff
@@ -290,6 +290,7 @@ void photoply_state::photoply(machine_config &config)
 		.erase_all_time(attotime::from_usec(10));
 }
 
+// We asume that every Photo Play from 1999 onwards use a DX4 100MHz instead of a 75MHz one (both were compatible, the later were recommended)
```

Comment:
“Later” and “latter” are not the same word.

Reply:
Fixed, thanks